src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -296,6 +299,16 @@ class StubGenerator: public StubCodeGenerator {
     __ fsd(f26, f26_save);
     __ fsd(f27, f27_save);
 
+    __ frrm(t0);
+    __ sd(t0, frm_save);
+    // Set fcsr to the state we need. We do want Round to Nearest. We
```

Comment:
I think it will be more accurate to mention `frm` instead of `fcsr` both in code comment and the JBS title?